docs/_static/css/custom.css

```diff
@@ -6,7 +6,7 @@ body.wy-body-for-nav {
     -webkit-font-smoothing: antialiased; /* Do not remove! */
```

Comment:
I have some doubts about antialiasing: it makes font thinner in webkit browsers (see screenshots), but I didn't remove it because of the `/* Do not remove! */` comment.

<img width="720" alt="without_antialiasing" src="https://user-images.githubusercontent.com/509198/35523430-bfdfcf7c-0516-11e8-8da1-74623579096c.png">

<img width="725" alt="antialiasing" src="https://user-images.githubusercontent.com/509198/35523451-cd756034-0516-11e8-82a6-0c7aa706c83a.png">